install_config/aggregate_logging.adoc

```diff
@@ -454,19 +462,14 @@ documentation] for more information on Kibana.
 
 === Cleanup
 
-After deployment, the deployer can be removed:
-
```

Comment:
Given we're probably going to reuse the account and secret for upgrades, I think it's best to leave these around.